<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Rabbit Remote Control</string>
  </property>
  <property name="windowIcon">
   <iconset resource="Resource/Resource.qrc">
    <normaloff>:/image/App</normaloff>:/image/App</iconset>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp_H">
    <property name="title">
     <string>Help(&amp;H)</string>
    </property>
    <addaction name="actionAbout_A"/>
    <addaction name="actionUpdate_U"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View(&amp;V)</string>
    </property>
    <widget class="QMenu" name="menuSink">
     <property name="title">
      <string>Sink</string>
     </property>
     <property name="icon">
      <iconset resource="Resource/Resource.qrc">
       <normaloff>:/image/Skin</normaloff>:/image/Skin</iconset>
     </property>
     <addaction name="actionDefaultStyle_D"/>
     <addaction name="actionOpenStyle_O"/>
    </widget>
    <widget class="QMenu" name="menuZoom">
     <property name="title">
      <string>Zoom</string>
     </property>
     <property name="icon">
      <iconset resource="Resource/Resource.qrc">
       <normaloff>:/image/Zoom</normaloff>:/image/Zoom</iconset>
     </property>
     <addaction name="actionZoom_window_to_remote_desktop"/>
     <addaction name="actionZoomToWindow_Z"/>
     <addaction name="actionKeep_aspect_ration_to_windows_K"/>
     <addaction name="actionOriginal_O"/>
     <addaction name="actionZoom_In"/>
     <addaction name="actionZoom_Out"/>
    </widget>
    <addaction name="actionFull_screen_F"/>
    <addaction name="menuZoom"/>
    <addaction name="separator"/>
    <addaction name="actionShow_TabBar_B"/>
    <addaction name="separator"/>
    <addaction name="actionScreenshot"/>
    <addaction name="actionSend_ctl_alt_del"/>
    <addaction name="menuSink"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="actionToolBar_T"/>
    <addaction name="actionStatusBar_S"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuRemote">
    <property name="title">
     <string>Remote(&amp;R)</string>
    </property>
    <widget class="QMenu" name="menuConnect_C">
     <property name="title">
      <string>Connect(&amp;C)</string>
     </property>
     <property name="icon">
      <iconset resource="Resource/Resource.qrc">
       <normaloff>:/image/Connect</normaloff>:/image/Connect</iconset>
     </property>
    </widget>
    <addaction name="menuConnect_C"/>
    <addaction name="actionDisconnect_D"/>
    <addaction name="actionOpen_O"/>
    <addaction name="actionClone"/>
    <addaction name="actionRecently_connected"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_favorite"/>
    <addaction name="actionFavorites"/>
    <addaction name="separator"/>
    <addaction name="actionCurrent_connect_parameters"/>
    <addaction name="actionSettings"/>
    <addaction name="separator"/>
    <addaction name="actionExit_E"/>
   </widget>
   <addaction name="menuRemote"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp_H"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionOpen_O"/>
   <addaction name="actionDisconnect_D"/>
   <addaction name="separator"/>
   <addaction name="actionFull_screen_F"/>
   <addaction name="separator"/>
   <addaction name="actionShow_TabBar_B"/>
   <addaction name="actionScreenshot"/>
   <addaction name="actionCurrent_connect_parameters"/>
   <addaction name="actionSettings"/>
   <addaction name="separator"/>
   <addaction name="actionExit_E"/>
  </widget>
  <action name="actionAbout_A">
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/image/App</normaloff>:/image/App</iconset>
   </property>
   <property name="text">
    <string>About(&amp;A)</string>
   </property>
   <property name="iconText">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
   <property name="statusTip">
    <string>About</string>
   </property>
   <property name="whatsThis">
    <string>About</string>
   </property>
  </action>
  <action name="actionUpdate_U">
   <property name="text">
    <string>Update(&amp;U)</string>
   </property>
   <property name="iconText">
    <string>Update</string>
   </property>
   <property name="toolTip">
    <string>Update</string>
   </property>
   <property name="statusTip">
    <string>Update</string>
   </property>
   <property name="whatsThis">
    <string>Update</string>
   </property>
  </action>
  <action name="actionStatusBar_S">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>StatusBar(&amp;S)</string>
   </property>
   <property name="iconText">
    <string>StatusBar</string>
   </property>
   <property name="toolTip">
    <string>StatusBar</string>
   </property>
   <property name="statusTip">
    <string>StatusBar</string>
   </property>
   <property name="whatsThis">
    <string>StatusBar</string>
   </property>
  </action>
  <action name="actionToolBar_T">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>ToolBar(&amp;T)</string>
   </property>
   <property name="iconText">
    <string>ToolBar</string>
   </property>
   <property name="toolTip">
    <string>ToolBar</string>
   </property>
   <property name="statusTip">
    <string>ToolBar</string>
   </property>
   <property name="whatsThis">
    <string>ToolBar</string>
   </property>
  </action>
  <action name="actionFull_screen_F">
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/image/FullScreen</normaloff>:/image/FullScreen</iconset>
   </property>
   <property name="text">
    <string>Full screen(&amp;F)</string>
   </property>
   <property name="iconText">
    <string>Full screen</string>
   </property>
   <property name="toolTip">
    <string>Full screen</string>
   </property>
   <property name="statusTip">
    <string>Full screen</string>
   </property>
   <property name="whatsThis">
    <string>Full screen</string>
   </property>
  </action>
  <action name="actionOriginal_O">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/image/Original</normaloff>:/image/Original</iconset>
   </property>
   <property name="text">
    <string>Original size(&amp;O)</string>
   </property>
   <property name="iconText">
    <string>Original size</string>
   </property>
   <property name="toolTip">
    <string>Original size</string>
   </property>
   <property name="statusTip">
    <string>Original size</string>
   </property>
   <property name="whatsThis">
    <string>Original size</string>
   </property>
  </action>
  <action name="actionZoomToWindow_Z">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/image/ZoomToWindow</normaloff>:/image/ZoomToWindow</iconset>
   </property>
   <property name="text">
    <string>Zoom to window(&amp;Z)</string>
   </property>
   <property name="iconText">
    <string>Zoom to window</string>
   </property>
   <property name="toolTip">
    <string>Zoom to window</string>
   </property>
   <property name="statusTip">
    <string>Zoom to window</string>
   </property>
   <property name="whatsThis">
    <string>Zoom to window</string>
   </property>
  </action>
  <action name="actionKeep_aspect_ration_to_windows_K">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/image/AspectRatio</normaloff>:/image/AspectRatio</iconset>
   </property>
   <property name="text">
    <string>Keep aspect ration zoom to window(&amp;K)</string>
   </property>
   <property name="iconText">
    <string>Keep aspect ration zoom to window</string>
   </property>
   <property name="toolTip">
    <string>Keep aspect ration zoom to window</string>
   </property>
   <property name="statusTip">
    <string>Keep aspect ration zoom to window</string>
   </property>
   <property name="whatsThis">
    <string>Keep aspect ration zoom to window</string>
   </property>
  </action>
  <action name="actionExit_E">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/image/Exit</normaloff>:/image/Exit</iconset>
   </property>
   <property name="text">
    <string>Exit(&amp;E)</string>
   </property>
   <property name="iconText">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
   <property name="statusTip">
    <string>Exit</string>
   </property>
   <property name="whatsThis">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRecently_connected">
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/image/RecentMenu</normaloff>:/image/RecentMenu</iconset>
   </property>
   <property name="text">
    <string>Recently connected</string>
   </property>
   <property name="statusTip">
    <string>Recently connected</string>
   </property>
   <property name="whatsThis">
    <string>Recently connected</string>
   </property>
  </action>
  <action name="actionDisconnect_D">
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/image/Disconnect</normaloff>:/image/Disconnect</iconset>
   </property>
   <property name="text">
    <string>Disconnect(&amp;D)</string>
   </property>
   <property name="iconText">
    <string>Disconnect</string>
   </property>
   <property name="toolTip">
    <string>Disconnect</string>
   </property>
   <property name="statusTip">
    <string>Disconnect</string>
   </property>
   <property name="whatsThis">
    <string>Disconnect</string>
   </property>
  </action>
  <action name="actionOpen_O">
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/image/Open</normaloff>:/image/Open</iconset>
   </property>
   <property name="text">
    <string>Open(&amp;O)</string>
   </property>
   <property name="iconText">
    <string>Open</string>
   </property>
   <property name="toolTip">
    <string>Open</string>
   </property>
   <property name="statusTip">
    <string>Open</string>
   </property>
   <property name="whatsThis">
    <string>Open</string>
   </property>
  </action>
  <action name="actionDefaultStyle_D">
   <property name="text">
    <string>Default(&amp;D)</string>
   </property>
   <property name="iconText">
    <string>Default style</string>
   </property>
   <property name="toolTip">
    <string>Default style</string>
   </property>
   <property name="statusTip">
    <string>Default style</string>
   </property>
   <property name="whatsThis">
    <string>Default style</string>
   </property>
  </action>
  <action name="actionOpenStyle_O">
   <property name="text">
    <string>Open(&amp;O)</string>
   </property>
   <property name="iconText">
    <string>Open style</string>
   </property>
   <property name="toolTip">
    <string>Open style</string>
   </property>
   <property name="statusTip">
    <string>Open style</string>
   </property>
   <property name="whatsThis">
    <string>Open style</string>
   </property>
  </action>
  <action name="actionSend_ctl_alt_del">
   <property name="text">
    <string>Send Ctl+Alt+Del</string>
   </property>
  </action>
  <action name="actionShow_TabBar_B">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/image/TabBar</normaloff>:/image/TabBar</iconset>
   </property>
   <property name="text">
    <string>Show TabBar(&amp;B)</string>
   </property>
   <property name="iconText">
    <string>Show TabBar(&amp;B)</string>
   </property>
   <property name="toolTip">
    <string>Show TabBar</string>
   </property>
   <property name="statusTip">
    <string>Show TabBar</string>
   </property>
   <property name="whatsThis">
    <string>Show TabBar</string>
   </property>
  </action>
  <action name="actionZoom_In">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/image/ZoomIn</normaloff>:/image/ZoomIn</iconset>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
   <property name="statusTip">
    <string>Zoom In</string>
   </property>
   <property name="whatsThis">
    <string>Zoom In</string>
   </property>
  </action>
  <action name="actionZoom_Out">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/image/ZoomOut</normaloff>:/image/ZoomOut</iconset>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
   <property name="statusTip">
    <string>Zoom Out</string>
   </property>
   <property name="whatsThis">
    <string>Zoom Out</string>
   </property>
  </action>
  <action name="actionZoom_window_to_remote_desktop">
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/image/ZoomWindowToRemoteDesktop</normaloff>:/image/ZoomWindowToRemoteDesktop</iconset>
   </property>
   <property name="text">
    <string>Zoom window to remote desktop</string>
   </property>
   <property name="statusTip">
    <string>Zoom window to remote desktop</string>
   </property>
   <property name="whatsThis">
    <string>Zoom window to remote desktop</string>
   </property>
  </action>
  <action name="actionScreenshot">
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/image/Screenshot</normaloff>:/image/Screenshot</iconset>
   </property>
   <property name="text">
    <string>Screenshot</string>
   </property>
   <property name="toolTip">
    <string>Screenshot</string>
   </property>
   <property name="statusTip">
    <string>Screenshot</string>
   </property>
   <property name="whatsThis">
    <string>Screenshot</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/image/Option</normaloff>:/image/Option</iconset>
   </property>
   <property name="text">
    <string>Settings</string>
   </property>
   <property name="statusTip">
    <string>Settings</string>
   </property>
   <property name="whatsThis">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionCurrent_connect_parameters">
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/image/Parameters</normaloff>:/image/Parameters</iconset>
   </property>
   <property name="text">
    <string>Current connect parameters</string>
   </property>
   <property name="toolTip">
    <string>Current connect parameters</string>
   </property>
   <property name="statusTip">
    <string>Current connect parameters</string>
   </property>
   <property name="whatsThis">
    <string>Current connect parameters</string>
   </property>
  </action>
  <action name="actionClone">
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/image/Clone</normaloff>:/image/Clone</iconset>
   </property>
   <property name="text">
    <string>Clone current connect</string>
   </property>
   <property name="statusTip">
    <string>Clone current connect</string>
   </property>
   <property name="whatsThis">
    <string>Clone current connect</string>
   </property>
  </action>
  <action name="actionFavorites">
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/image/Favorites</normaloff>:/image/Favorites</iconset>
   </property>
   <property name="text">
    <string>Favorites</string>
   </property>
   <property name="statusTip">
    <string>Favorites</string>
   </property>
   <property name="whatsThis">
    <string>Favorites</string>
   </property>
  </action>
  <action name="actionAdd_favorite">
   <property name="icon">
    <iconset resource="Resource/Resource.qrc">
     <normaloff>:/image/AddFavorites</normaloff>:/image/AddFavorites</iconset>
   </property>
   <property name="text">
    <string>Add favorite</string>
   </property>
   <property name="statusTip">
    <string>Add favorite</string>
   </property>
   <property name="whatsThis">
    <string>Add favorite</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resource/Resource.qrc"/>
 </resources>
 <connections/>
</ui>
